wrappers/dlltrace.hpp

```diff
@@ -28,6 +28,8 @@
 
 #include <windows.h>
 #include <string>
+#include <cstdlib>
```

Comment:
I prefer we don't use C++ variants of C headers, because they are often unreliable.  Otherwise looks good.